README.md

```diff
@@ -79,6 +79,7 @@ If you'd like to change something:
 | `COLOR_WARNING`                          | Change default output color for Warning/<br>Interaction needed messages. [Available colors](https://github.com/pdffiller/styleguide-hooks/blob/stable/hooks/color_echo.sh).            | `"Yellow"`            | `pre-commit`                          |
 | `IGNORE_TRAILING_`<br>`SPACE_EXTENTIONS` | Ignore trailing whitespaces for specified file extentions. Format `ext1\|ext2\|ext3`.                                                                                                  | `"md\|rst"`           | `pre-commit`                          |
 | `INTERACTIVE_MODE`                       | Pseudo-interactive mode for:<br>1. Replacing trailing whitespaces<br>2. Removing multilines at EOF<br>3. Adding new line at EOF if it doesn't exist<br>Can be `disabled` or `enabled`. | `"disabled"`          | `pre-commit`                          |
+| `FORSE_WARNING_TO_ERROR`                 | Forsing exit if enabled.Affects next checks:<br>1. Too many changes per one commit<br>Can be `disabled` or `enabled`.                                                                  | `"disabled"`          | `pre-commit`                          |
```

Comment:
~FORSE~ --> FORCE
> Forsing exit if enabled.Affects next checks:<br>1. Too many changes per one commit<br>Can be `disabled` or `enabled`.  

Fix spelling here. Also, list is intended but has only one item. Thus it will be better to write as follows:  

> Affects "Too many changes per one commit" check.<br>Can be `disabled` or `enabled`.